### PR TITLE
fix(dynamic-branding) Fix bogus mUI dependency on mobile

### PR DESCRIPTION
**Pull request by horymury**

### CODE REVIEW

File: react/features/dynamic-branding/middleware.web.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { MiddlewareRegistry } from '../base/redux';
```
Comment (saghul) [inline react/features/dynamic-branding/middleware.web.js:5]: If you are renaming this file you'll need to adjust the import in middlewares.js

Comment (horymury) [reply react/features/dynamic-branding/middleware.web.js:5]: I tested this and it worked, I see that we already have it in middewares.web.js. I guess I could either remove the middleware.native.js completely as it is empty, or add it in the middlewares.native.js and it should take the empty one.

Comment (horymury) [reply react/features/dynamic-branding/middleware.web.js:5]: Shall I just remove middlewares.native.js completely? WDYT?

Comment (saghul) [reply react/features/dynamic-branding/middleware.web.js:5]: You can remove it then.

**Comment by horymury** [comment]
Jenkins please test this please.

**Review by Calinteodor — APPROVED** [review]

**Comment by horymury** [comment]
Jenkins please test this please.

**Pull request merged**